Essentials/src/main/java/com/earth2me/essentials/commands/Commandwhois.java

```diff
@@ -40,6 +41,7 @@ public void run(final Server server, final CommandSource sender, final String co
         sender.sendMessage(tl("whoisHunger", user.getBase().getFoodLevel(), user.getBase().getSaturation()));
         sender.sendMessage(tl("whoisExp", SetExpFix.getTotalExperience(user.getBase()), user.getBase().getLevel()));
         sender.sendMessage(tl("whoisLocation", user.getLocation().getWorld().getName(), user.getLocation().getBlockX(), user.getLocation().getBlockY(), user.getLocation().getBlockZ()));
+        sender.sendMessage(tl("whoisJoined", DateUtil.formatDateDiff(user.getBase().getFirstPlayed()), new SimpleDateFormat("yyyy/MM/dd, HH:mm").format(user.getBase().getFirstPlayed())));
```

Comment:
I think it would be nicer visually to place this right above the UUID as done in seen. That way the outputs match a bit more closely. Arguably this could also be done for the IP.